python/ray/_private/worker.py

```diff
@@ -2284,6 +2288,18 @@ def get(
         Exception: An exception is raised if the task that created the object
             or that created one of the objects raised an exception.
     """
+    if timeout == 0:
+        import warnings
+
+        warnings.warn(
+            (
+                "Please use timeout=None if you expect ray.get() to block. "
```

Comment:
can you add an env var to suppress the warning? Or guide users how to do it using warnings.filterwarning